src/components/Settings/Settings.tsx

```diff
@@ -84,6 +99,7 @@ const Settings = () => {
                 minLength={1}
                 value={deviceNickname}
                 onChange={(e) => setDeviceNickname(e.target.value)}
+                disabled={activeNode ? false : true}
```

Comment:
Nit: could also use `!activeNode` which converts to `true` only if `activeNode` is `null`